AdyenDropIn/DropInComponent.swift

```diff
@@ -220,9 +220,7 @@ public final class DropInComponent: NSObject,
         case let component as PresentableComponent where component.requiresModalPresentation:
             navigationController.present(asModal: component)
```

Comment:
BTW `present(asModal:)` also checks `if component.requiresModalPresentation `. Could be redundant..